modules/tutorials/pages/migration-tutorial/sql-migration-tutorial-capella.adoc

```diff
@@ -0,0 +1,236 @@
+= Migrating your Data from MySQL to Capella
```

Comment:
I'm a bit confused about having the Capella tutorial here, in the `release/7.6` branch.

Shouldn't just the Server version be here,
with the Capella one in the `capella` branch of `docs-devex`,
and them having equivalent places in the nav for their respective branches?